### DISCUSSION TITLE
Add support for Times Gate

### DISCUSSION THREAD

**Discussion by forgottenruntime** [gickowtf/pixoo-homeassistant#13] [Ideas]
Hi there and thanks for your effort you put in to this . It's realy good . 

Do you have the posibility to add support for Times Gate ? 
https://divoom.com/products/time-gate

**Comment by SB2DD** [comment]
Well, the API is VERY similar, so it would be very possible, but I don't know if Gicko has one for testing (I personally don't).

**Comment by gickowtf** [comment]
@SpeedlimitsB , Unfortunately, I don't have timesgate either. As @Mrredstone5230 says, it would be good to test this.
